.zenodo.json

```diff
@@ -1,7 +1,7 @@
 {
-    "description": "The community-developed, free and open-source solar data analysis environment for Python.",
+    "description": "sunpy is a software library that provides fundamental tools for accessing, loading and interacting with solar physics data in Python",
     "license": "BSD-2-Clause",
-    "title": "SunPy",
+    "title": "SunPy Core Package: sunpy",
```

Comment:
I don't think this is going to work for anyone other than the 4 of us who get picky about `sunpy` vs `SunPy`. I think this should be `The SunPy Core Package` or `sunpy: A Core Package for Solar Physics` or something